controllers/managedocs_controller.go

```diff
@@ -77,6 +77,7 @@ func (r *ManagedOCSReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	err := r.reconcilePhases(req)
 
 	// Ensure status is updated once even on failed reconciles
+	r.managedOCS.Status.ReconcileCount++
```

Comment:
Not needed

Reply:
I'm using it for testing and debugging purposes. Many of the tests require a reconciliation loop to occur before checking if reconcile did what we expected. Plus, I'm interested to know how many times the CR is being reconciled.

Reply:
Please see my comment about public API above. You can store something on the reconciler or the manager if needed.

Reply:
Keeping track of the times the cr was reconciled provides an immediate way for the tests to know when they can check what they need to.

Reply:
The last update time is already on the CR and is part of the public API. Please use it instead

Reply:
Where?

Reply:
The create/update count is captured by `resourceVersion` and the timestamp should be captured somewhere in the metadata (I beleive under ManagedFields). But if not we can add it to the spec - this will be reasonable for a public API